tensorflow/lite/tools/test_utils.py

```diff
@@ -117,7 +117,7 @@ def build_mock_flatbuffer_model():
   schema_fb.TensorStart(builder)
   schema_fb.TensorAddName(builder, string1_offset)
   schema_fb.TensorAddShape(builder, shape1_offset)
-  schema_fb.TensorAddType(builder, 0)
+  schema_fb.TensorAddType(builder, 3) #UINT8
```

Comment:
for what?

Reply:
Got it. Then `schema_fb.TensorAddType(builder, TensorType_UINT8)` ?

Reply:
Yes, using `TensorType_UINT8` is better. I'll update the code. Thanks!